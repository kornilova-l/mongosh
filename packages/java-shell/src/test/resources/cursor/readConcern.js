// before
db.coll.drop();
db.coll.insertOne({a: 1});
// command
db.coll.find().readConcern('local');
// command
db.coll.find().readConcern('unknown');
// clear
db.coll.drop();