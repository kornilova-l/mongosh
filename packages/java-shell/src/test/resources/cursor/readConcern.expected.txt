[ { "_id": <ObjectID>, "a": 1 } ]